### DISCUSSION TITLE
key remapping and click here to continue

### DISCUSSION THREAD

**Discussion by Dasbi** [runelite/runelite#17299] [Ideas]
key remapping doesnt work the same as F-keys when a "click here to continue" box pops up, is there any way this could work the same way fkeys work? thanks!

for more clarification, if you have remapped F-keys and you have them rebound to a letter on your keyboard and you get a click here to continue over the chat box then clicking the remapped key doesnt open the interface its tied to, it does nothing.

**Comment by tyleralbee** [comment]
I am pretty sure this is the same thing I was looking to feature request so I'll comment here.

When the **Key Remapping** plugin is used to **Remap F Keys**, you're no longer able to buffer inputs for various interfaces. For example, if I hold `1` and click on a furnace, the input will not be buffered - that is, I would have to click `1` again to select the first option.

I can see that the issue #11578 suggested that this was a regression and led to a feature request fixing the issue. The example used in that issue was holding 1 before clicking a spirit tree, which does work in the way I'm suggesting (i.e. the `1` input is buffered).

I would be willing to go through in-game finding interfaces that input buffering is not working for. I would also be willing to work towards fixing this issue myself if someone were willing to lend a hand pointing me in the right direction.
